src/IB/IBFEMethod.cpp

```diff
@@ -535,6 +508,21 @@ IBFEMethod::setupTagBuffer(Array<int>& tag_buffer, Pointer<GriddingAlgorithm<NDI
     return;
 } // setupTagBuffer
 
+double
+IBFEMethod::getMaxPointDisplacement() const
+{
+    double max_displacement = 0.0;
+    for (unsigned int part = 0; part < d_meshes.size(); ++part)
+    {
+        PetscVector<double>& last = d_X_vecs->get("last_regrid", part);
+        PetscVector<double>& diff = d_X_vecs->get("tmp", part);
+        diff = d_X_vecs->get("solution", part);
+        diff -= last;
+        max_displacement = std::max(max_displacement, diff.linfty_norm());
+    }
+    return max_displacement / get_min_patch_dx(d_hierarchy, getFinestPatchLevelNumber());
```

Comment:
Not a common use case, but for stretched grids, could we benefit from handling this dimension-by-dimension?

Reply:
In principle yes and this wouldn't be that hard to do (we just need to subdivide the finite element vector into x, y, and z components) but I don't think its worth doing now since we have no such examples (and I have no idea what else might break in the library if we do this).